lib/base64.nit

```diff
@@ -34,13 +34,9 @@ redef class String
 	end
 
 	# Encodes the receiver string to base64.
-	# By default, uses "=" for padding.
-	fun encode_base64 : String do return encode_base64_custom_padding('='.ascii.to_b)
-
-	# Encodes the receiver string to base64 using a custom padding character.
 	#
-	# If using the default padding character `=`, see `encode_base64`.
-	fun encode_base64_custom_padding(padding : Byte) : String
+	# If no padding is given, will use `=` as padding the character
```

Comment:
Need some nitunits like:

~~~nit
# Encodes the receiver string to base64.
# 
# If no padding is given, will use `=` as padding the character
#
#     assert "".encode_base64 == ""
#     assert "nit is lame".encode_base64 == "bml0IGlzIGxhbWU="
#     assert "nit is lame".encode_base64(45.to_b) == "bml0IGlzIGxhbWU-"
~~~